imdb/UI/View/ProfileViewController.swift

```diff
@@ -9,9 +9,204 @@ import Foundation
+        
+        view.addSubview(titleLabel)
+        view.addSubview(imageView)
+        
+        titleLabel.translatesAutoresizingMaskIntoConstraints = false
+        imageView.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate([
+            titleLabel.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor, constant: 44),
+            titleLabel.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 8),
+            titleLabel.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -24),
+            titleLabel.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+            
+            imageView.topAnchor.constraint(equalTo: titleLabel.bottomAnchor, constant: 29),
+            imageView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 32),
+            imageView.trailingAnchor.constraint(equalTo: view.safeAreaLayoutGuide.trailingAnchor)
+        ])
+        
+        
+        let textView = UITextView()
+        textView.isEditable = false
+        textView.isScrollEnabled = false
+        textView.textContainerInset = .zero
+        textView.textContainer.lineFragmentPadding = 0
+        textView.backgroundColor = .clear
+        textView.font = UIFont(name: "SF Pro Display", size: 24)
+        textView.textAlignment = .left
+        
+        let line1Text = "⦿ Access more than 1,000,000 people reviews ⭐\n"
+        let line2Text = "⦿ Save your library 📚\n"
+        let line3Text = "⦿ Share your thoughts 🎤"
+        let bulletText = "\(line1Text)\(line2Text)\(line3Text)"
+        let attributedString = NSMutableAttributedString(string: bulletText)
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.tabStops = [NSTextTab(textAlignment: .left, location: 20, options: [:])]
+        paragraphStyle.defaultTabInterval = 20
+        let attributes: [NSAttributedString.Key: Any] = [
+            .foregroundColor: UIColor.black,
+            .font: textView.font ?? UIFont.systemFont(ofSize: 24),
+            .paragraphStyle: paragraphStyle
+        ]
+        attributedString.addAttributes(attributes, range: NSRange(location: 0, length: attributedString.length))
+        textView.attributedText = attributedString
+        
+        view.addSubview(textView)
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        
+        NSLayoutConstraint.activate([
+            textView.topAnchor.constraint(equalTo: imageView.bottomAnchor, constant: 29),
+            textView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 20),
+            textView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: -20),
+            textView.bottomAnchor.constraint(lessThanOrEqualTo: view.bottomAnchor, constant: -20)
+        ])
+        
+        textView.sizeToFit()
+        textView.layoutIfNeeded()
+        
+        
+        let signInButton = UIButton()
```

Comment:
You have very similar buttons. Maybe you can create new button with main attributes and make easier configuration?